fuse_core/include/fuse_core/graph_deserializer.h

```diff
@@ -89,6 +89,7 @@ class GraphDeserializer
 private:
   pluginlib::ClassLoader<fuse_core::Variable> variable_loader_;  //!< Pluginlib class loader for Variable types
   pluginlib::ClassLoader<fuse_core::Constraint> constraint_loader_;  //!< Pluginlib class loader for Constraint types
+  pluginlib::ClassLoader<fuse_core::Constraint> loss_loader_;  //!< Pluginlib class loader for Loss types
```

Comment:
I assume this should be:
```suggestion
  pluginlib::ClassLoader<fuse_core::Loss> loss_loader_;  //!< Pluginlib class loader for Loss types
```

Reply:
Fixed